app/components/GameOverlaySettings.tsx

```diff
@@ -19,7 +19,16 @@ export default class GameOverlaySettings extends TsxComponent<{}> {
 
   async setEnableGameOverlay(val: boolean) {
     this.enabling = true;
-    await this.gameOverlayService.setEnabled(val);
+    try {
+      await this.gameOverlayService.setEnabled(val);
+    } catch (e) {
```

Comment:
You should check the type of `e` here.  If `setEnabled` raises an exception this will be an exception.  I'm not sure how that will serialize or what toasted will do with it, but my guess is it will be jank.